backend/app/services/chat/trigger/core.py

```
@@ -1837,3 +1858,115 @@ def _get_bot_mcp_servers_for_http(bot_name: str, bot_namespace: str) -> Dict[str
+            subtask=assistant_subtask,
+            team=team,
+            user=user,
+            auth_token=auth_token,
+            user_subtask=user_subtask,
+        )
+        logger.info(
+            "[ai_trigger] Task successfully routed to device: task_id=%d, device_id=%s",
+            task.id,
+            device_id,
+        )
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)